requirements.txt

```diff
@@ -1,4 +1,3 @@
 click
 requests
 pyyaml
-statusdb
```

Comment:
I never tried `pip` to test the install, but I guess it would work without any extra parameters :sweat_smile: